pkg/util/kubernetes/hostinfo/no_apiserver.go

```diff
@@ -3,8 +3,8 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-present Datadog, Inc.
 
-//go:build !kubeapiserver
-// +build !kubeapiserver
+//go:build kubelet && !kubeapiserver
```

Comment:
Why do we need the `kubelet` tag?

Reply:
When running with the IoT Agent build tags, `golangci-lint` complained with:
```
running golangci on ./pkg
pkg/util/kubernetes/hostinfo/no_apiserver.go:13:6: `apiserverNodeLabels` is unused (deadcode)
func apiserverNodeLabels(ctx context.Context, nodeName string) (map[string]string, error) {
     ^
pkg/util/kubernetes/hostinfo/no_apiserver.go:17:6: `apiserverNodeAnnotations` is unused (deadcode)
func apiserverNodeAnnotations(ctx context.Context, nodeName string) (map[string]string, error) {
     ^
 ```